providers/curation/github.js

```
@@ -36,6 +35,40 @@ class GitHubCurationService {
     }
   }
 
+  prOpened(pr) {
+    return this._storeAndValidateContribution(pr)
+  }
+
+  prClosed(pr) {
+    return this.store.updateContribution(pr)
+  }
+
+  async prMerged(pr) {
+    // update the merged PR. Don't need to store the proposed changes as they should already be there.
+    await this.store.updateContribution(pr)
+
+    // get and store the current state of the curation branch
+    const curations = await this._getContributedCurations(pr.number, pr.head.sha)
+    await this.store.updateCurations(curations)
+
+    // invalidate all affected definitions then recompute. This ensures the changed defs are cleared out
+    // even if there are errors recomputing the definitions.
+    const coordinateList = Curation.getAllCoordinates(curations)
+    await this.definitionService.invalidate(coordinateList)
+    return Promise.all(coordinateList.map(coordinates => this.definitionService.computeAndStore(coordinates)))
```

throttle this, maybe let them fail individually